wp/lib/bap_wp/src/environment.ml

```diff
@@ -36,6 +37,7 @@ type t = {
   precond_map : Constr.t TidMap.t;
   fun_name_tid : Tid.t StringMap.t;
   call_map : string TidMap.t;
+  fun_outputs_map : FuncDecl.func_decl list TidMap.t;
```

Comment:
Now I wonder if we should have comments explaining all these fields...

Reply:
Gonna remove this field from the `env`.